### ISSUE TITLE
Fail to load the package in the virtualenv 

### ISSUE THREAD

**Issue by rylanchiu** [yhat/rodeo#240]
I have set the python path already as the doc said: `/Users/yobichi/hp/bin/python`. But when I import the `pandas ` package, an error `ImportError: No module named pandas` occurred. And when I tried to install  that package, `pip: command not found`. Could you please tell me how to solve this?

**Comment by incredibeau** [comment]
Same issue here. I can't load packages. For example, I installed beautifulsoup4 from Terminal, but in Rodeo when I run `import beautifulsoup4` I get the error `ImportError: No module named 'beautifulsoup4'`

**Comment by johanovic** [comment]
Indeed, the only workaround I've found is to set the lib/python/site-packages directory as the 'working directory' in Rodeo. That does load all the site packages in the venv.
